.gitignore

```diff
@@ -7,3 +7,4 @@
 platformio-device-monitor*.log
 platformio_override.ini
 .DS_Store
+webapp_dist
```

Comment:
Will not add this as it's makes things hard to distribute pre-compiled binaries.

Reply:
removed ignore